modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -251,7 +250,7 @@ export default class Checkbox extends React.Component<CheckboxProps> {
           <CheckboxInput
             checked={checked}
             disabled={disabled}
-            id={id}
+            id={id || this.id}
```

Comment:
Should we add a story with multiple checboxes? @NicholasBoll

Reply:
Nope, but we should add a test for it @mannycarrera4